:lipstick: Advance architecture docs (#11074)
DefectDojo/django-DefectDojo: docs/content/en/getting_started/architecture.md

## Message Broker

> The application server sends tasks to a [Message Broker](https://docs.celeryq.dev/en/stable/)

Reviewer: To match the original page link

```suggestion
The application server sends tasks to a [Message Broker](https://docs.celeryq.dev/en/stable/getting-started/backends-and-brokers/index.html)
```

> for asynchronous execution. The celery currently only supports to use [Redis](https://redis.io/solutions/messaging/) as a broker. 

Reviewer: ```suggestion
for asynchronous execution. Currently, only [Redis](https://github.com/redis/redis) is supported as a broker.
```
